src/live_restore/live_restore_fs.c

```diff
@@ -1229,6 +1229,62 @@ __wt_live_restore_fh_to_metadata(WT_SESSION_IMPL *session, WT_FILE_HANDLE *fh, W
     return (ret);
 }
 
+/*
+ * __wt_live_restore_clean_metadata_string --
+ *     This function is only called when taking a backup. It validates the live restore metadata
+ *     string, either aborting on an unrecoverable error or cleaning up an outdated config value.
+ */
+int
+__wt_live_restore_clean_metadata_string(WT_SESSION_IMPL *session, char *value)
+{
+    WT_CONFIG_ITEM v;
+    WT_DECL_RET;
+
+    WT_ASSERT_ALWAYS(session, !F_ISSET(S2C(session), WT_CONN_LIVE_RESTORE_FS),
```

Comment:
This isn't guaranteed to be true, we never fully disable live restore. So in theory a backup could be taken on a live restore system.

Reply:
The `open_cursor` logic bans backups whenever `WT_CONN_LIVE_RESTORE_FS` is set [here](https://github.com/wiredtiger/wiredtiger/blob/3c351577aaa17c93bee2e5aa0e43539ac1fc8f83/src/session/session_api.c#L696). Do we need a follow on ticket to enable backups in the COMPLETE stage?

Reply:
Yes I think so

Reply:
Created and added FIXMEs for WT-14231 👍